electron/renderer/static/webview-preload.js

```diff
@@ -25,7 +25,17 @@ const winston = require('winston');
 const {EVENT_TYPE} = require('../../dist/lib/eventType');
 
 const {desktopCapturer, ipcRenderer, remote, webFrame} = require('electron');
-const {app} = remote;
+const {app, systemPreferences} = remote;
+
+// Note: Until appearance-changed event is available in a future
+// version of Electron... use AppleInterfaceThemeChangedNotification
+function appearanceManager() {
+  if (environment.platform.IS_MAC_OS) {
+    systemPreferences.subscribeNotification('AppleInterfaceThemeChangedNotification', () =>
+      amplify.publish(z.event.WebApp.PROPERTIES.UPDATE.APPEARANCE.SYSTEM_CHANGED, systemPreferences.isDarkMode())
```

Comment:
1. Event will be ignored
2. This can be translated from within the webapp

Reply:
1. If you call `z.event.WebApp.PROPERTIES.UPDATE.APPEARANCE.SYSTEM_CHANGED` in the wrapper but `z.event.WebApp.PROPERTIES.UPDATE.APPEARANCE` doesn't even exist, then you will receive a `Cannot call X of undefined` error, no?
2. It shouldn't be translated in the webapp. Wrapper should be able to give the right value in the first place.

Reply:
Let's name the event `z.event.WebApp.PROPERTIES.UPDATE.THEME`. 

This way we don't have problems when `THEME` doesn't exist (because `z.event.WebApp.PROPERTIES.UPDATE` is already there) and it matches perfectly our context of updating the theme. 

Our web app can then describe to it the following way:

```js
amplify.subscribe(z.event.WebApp.PROPERTIES.UPDATE.THEME, (name) => {
  console.log(`Applying "${name}" theme...`);
});
```